envs/unzip.yml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
Also can't find where this is referenced and again could just be missing it. What is the "rule" or guidance on how you should separate `.yml` files for environments and dependencies?

Reply:
i do it per snakemake rule bc i like to keep the environments tiny and fast. I don't think there is strict guidance though. I like per-tool when possible bc then there are fewer weird dependency solve things in the background that could silently change behavior or something